core/src/main/java/greencity/service/impl/UserServiceImpl.java

```diff
@@ -630,4 +634,18 @@ public void addDefaultHabit(Long userId, String language) {
         }
         return habitDictionaryDtos;
     }
+
+    /**
+     * Get profile picture path {@link String}.
+     *
+     * @return profile picture path {@link String}
+     */
+    @Override
+    public String getProfilePicturePathByUserId(Long id) {
```

Comment:
Where is this method used ? Maybe you shouldn't throw exception if picture path not found as user signs in for the first time - he won't have profile picture ?